docs/appendix/message-transport-mechanism.md

```diff
@@ -111,20 +123,10 @@ Assuming the Internet domain name of the remote participant is example.com, the
 
 [^B14]: P. Mockapetris, ""Domain names - concepts and facilities", STD 13, RFC 1034," 1987.
 
-The message will remain in the service discovery stage until all required data is available, or a participant-configurable and possibly message-class specific timeout timer expires.
-A DNS reply must have the NOERROR status, as well as syntactically valid and DNSSEC-authenticated content, in order to be considered usable.
-Other status codes such as SERVFAIL/NXDOMAIN, as well as connection failures or invalid record contents are considered temporary errors, in which case the lookup will be retried later within the timeout period.
-
 To prevent transient Internet connectivity or participant configuration issues from inhibiting message exchange, the timer value used for such a period should be at least one hour for routine messages.
 If service discovery fails due to inability to collect all required data within this timeout period, the delivery of the message fails.
 Such messages are removed from the outgoing queue and returned to the sending process, indicating a transport error.
 Note that, although message queue implementations may inform the sending process about delivery delays, only the final success or failure status is authoritative.
-When retrying failed requests, the implementation must implement an exponential back off mechanism to prevent undue load on the remote service.
```

Comment:
I think the sentences about retry and maximum delay interval should stay as they are general and not specific to DNS or any other discovery mechanism

Reply:
Moved it to a more generic spot in "Message transport mechanism"